### ISSUE TITLE
SB clearence with Rapido strain relief

### ISSUE THREAD

**Issue by DaLeberkasPepi** [VoronDesign/Voron-Afterburner#37]
My Rapido Hotend came with an included strain relief. Which i know the early review units didnt have. But thats also was causing problems for me. With the included strain relief i cant get the Rapido to fit into the cable channels.
Printed Parts:
[Stealthburner_toolhead_rapido_rear_cw2_beta0, Stealthburner_toolhead_rapido_front_beta0]

![IMG_20211229_163352](https://user-images.githubusercontent.com/1573408/147679477-5d27a98c-3a5c-4806-8012-73bced5da95d.jpg)
![IMG_20211229_163419](https://user-images.githubusercontent.com/1573408/147679522-759f183c-56b2-4fcc-bdf5-1f6e614bd06f.jpg)

My suggested solution would be to make a cutout at the side of the cable channel (like shown in the picture below)
![IMG_20211229_163822](https://user-images.githubusercontent.com/1573408/147679327-26cf7679-9295-4cc7-8fad-21f3a2426c9d.jpg)
![IMG_20211229_163801](https://user-images.githubusercontent.com/1573408/147679562-1626b9d4-dadb-46a5-b0c7-cff40414fcec.jpg)
![IMG_20211229_163813](https://user-images.githubusercontent.com/1573408/147679578-d0ba0be1-505e-43a4-8d66-88dab2e428a9.jpg)

**Comment by raymondh2** [comment]
This is already being worked on

**Comment by raymondh2** [comment]
@DaLeberkasPepi do the latest files fix the issue fo you?

**Comment by DaLeberkasPepi** [comment]
I'm in the process of reprinting that piece. I'll post pictures of it in place but the new version looks good.

**Comment by DaLeberkasPepi** [comment]
With the new version the rapido fits no matter what orientation it gets mounted in. 

At the beginning i didnt know that i could rotate the heatsink when i removed the 3 screws (I did think it would have been a too tight press fit or something) so i could have made it work by rotating it maybe... sry. 

But honestly i dont want anyone encourage to remove the 3 skrews to rotate the heater core only to tighten them wrong and bending the stainless steel heatbreak....

TLDR: Now it should work no matter how oriented your heater block gets mounted on rapido. Thanks wil.e and the Voron team for the great work!

![IMG_20220108_171353](https://user-images.githubusercontent.com/1573408/148651833-f5a3a421-be95-488c-889a-ac8b692d7a51.jpg)
![IMG_20220108_171344](https://user-images.githubusercontent.com/1573408/148651835-be9d1cbc-63b5-4338-9138-8dd3674a6845.jpg)
![IMG_20220108_171715](https://user-images.githubusercontent.com/1573408/148651836-81127c80-7995-4807-95ca-21ed0976235b.jpg)
![IMG_20220108_171135](https://user-images.githubusercontent.com/1573408/148651837-98782d8d-53f8-4773-8414-a813d4e423ac.jpg)
